v2/mfa/backend/second-factor.mdx

```diff
@@ -472,7 +472,52 @@ Passwordless.init({
 TODO
 </TabItem>
 <TabItem value="python">
-TODO
+
+```python
+from supertokens_python.recipe.passwordless.interfaces import APIInterface, APIOptions, ConsumeCodePostOkResult
+from typing import Union, Dict, Any
+from supertokens_python.recipe.session.asyncio import get_session
+from supertokens_python.recipe.usermetadata.asyncio import update_user_metadata
+
+
+def override_passwordless_apis(original_implementation: APIInterface):
+    original_consume_code_post = original_implementation.consume_code_post
+
+    async def consume_code_post(
+        pre_auth_session_id: str,
+        user_input_code: Union[str, None],
+        device_id: Union[str, None],
+        link_code: Union[str, None],
+        api_options: APIOptions,
+        user_context: Dict[str, Any],
+    ):
+        # this API is called when the user enters the OTP
+        # we should already have a session here since this is called
+        # after first factor login
+        _session = await get_session(api_options.request)
+        assert _session is not None
+
+        res = await original_consume_code_post(
+            pre_auth_session_id,
+            user_input_code,
+            device_id,
+            link_code,
+            api_options,
+            user_context,
+        )
+
+        if isinstance(res, ConsumeCodePostOkResult):
+            await update_user_metadata(
+                _session.get_user_id(), {"passwordlessUserId": res.user.user_id}
+            )
```

Comment:
Add a comment here like how there is for nodejs

Reply:
Make sure that all your code snippets have been commented as per the node snippets

Reply:
done.

Reply:
done. added missing comments.